### ISSUE TITLE
RSS Feed not working

### ISSUE THREAD

**Issue by helmikhalish** [Qrivi/RLRSS#5]
It says "Application error" when I check the RSS Feed.

**Comment by Qrivi** [comment]
This is a limitation of Heroku's free tier and not in my control. Too many people are polling the demo url which causes the Heroku account to run out of dynos and showing the "application error" -- as described [at the top of the readme](https://github.com/Qrivi/RLRSS/blob/master/README.md), actually. 😄

This issue should resolve itself tomorrow, August 1st, when the counters are reset. However if you plan on using RLRSS please deploy the code to your own Heroku account. Otherwise you'll just put more load on my account which will cause it to run out of dynos even faster.

I'll update the readme with a 1-click install to Heroku button later today. 

So tl;dr Heroku limitation, not a bug in the codebase.

**Comment by Qrivi** [comment]
Added! If you hit the button to deploy to your own Heroku instance, it'll start working straightaway and you should not run out of free dynos if you don't publicly share your url. 🙂

[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/Qrivi/RLRSS)
